File: intranet/LadoCliente/intranetMVC/Models/Alumno.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace intranetMVC.Models
{
    public class Alumno
    {
        public Alumno()
        {
            this.IdAlumno = 0;
            this.NomAlumno = string.Empty;
            this.ApeMatAlumno = string.Empty;
            this.ApePatAlumno = string.Empty;
            this.CodigoAlu = string.Empty;
            this.DNI = string.Empty;
            this.Carrera = string.Empty;
            this.EmailAlumno = string.Empty;
            this.Tipo = string.Empty;

        }

        public int IdAlumno { get; set; }

        public string ApePatAlumno { get; set; }

        public string ApeMatAlumno { get; set; }

        public string NomAlumno { get; set; }

        public string DNI { get; set; }

        public string CodigoAlu { get; set; }

        public string TelAlumno { get; set; }

        public char Sexo { get; set; }

        public string EmailAlumno { get; set; }

        public string DirAlumno { get; set; }

        public string Tipo { get; set; }

        public string Carrera { get; set; }
    }
}

File: intranet/LadoCliente/intranetMVC/Scripts/Alumno.js
window.onload = function () {
    ListarAlumnos();
};

function ListarAlumnos() {
    pintar(
        {
            url: "Alumno/getClientes",
            id: "divTabla",
            cabeceras: ["Id", "Nombre", "Ap. Paterno", "Ap. Materno", "DNI", "Teléfono"],
            propiedades: ["IdAlumno", "NomAlumno", "ApePatAlumno", "ApeMatAlumno", "DNI", "TelAlumno"],
            editar: true,
            eliminar: true,
            urlEliminar: "Alumno/Delete",
            parametroEliminar: "IdAlumno",
            urlRecuperar: "Alumno/AlumnoObtener",
            parametroRecuperar: "IdAlumno",
            propiedadId: "IdAlumno"
        },
        {
            busqueda: true,
            url: "Alumno/AlumnoBuscar",
            nombreparametro: "nombre",
            type: "text",
            button: true,
            id: "txtnombrealumno"
        },
        {
            id: "frmAlumno",
            type: "fieldset",
            urlGuardar: "Alumno/Create",
            legend: "Datos del alumno",
            formulario: [
                [
                    { class: "mb-3 col-md-7", label: "Nombre",           name: "NomAlumno",    classControl: "o max-50 min-3" },
                    { class: "mb-3 col-md-7", label: "Apellido Paterno", name: "ApePatAlumno", classControl: "o max-50 min-3" },
                    { class: "mb-3 col-md-7", label: "Apellido Materno", name: "ApeMatAlumno", classControl: "o max-50 min-3" },
                    { class: "mb-3 col-md-7", label: "DNI",              name: "DNI",          classControl: "o max-8 min-8 snc" },
                    { class: "mb-3 col-md-7", label: "Teléfono",         name: "TelAlumno",    classControl: "max-15 snc" },
                    { class: "mb-3 col-md-7", label: "Email",            name: "EmailAlumno",  classControl: "max-100" },
                    { class: "mb-3 col-md-7", label: "Dirección",        name: "DirAlumno",    classControl: "max-200" }
                ]
            ]
        }
    );
}
